docs/CODE_OF_CONDUCT-es.md

```diff
@@ -0,0 +1,78 @@
+# Código de Conducta del Pacto del Contribuyente
```

Comment:
```suggestion
# Código de Conducta Convenido para Contribuyentes
```
You can find the a translation in different languages for the "code of conduct" here https://www.contributor-covenant.org/translations/